packages/framework/src/Form/Constraints/FileExtensionMaxLengthValidator.php

```diff
@@ -9,7 +9,7 @@
 class FileExtensionMaxLengthValidator extends ConstraintValidator
 {
     /**
-     * @param string $value
+     * @param string|\Symfony\Component\HttpFoundation\File\File $value
```

Comment:
doesn't seem like `string`

Reply:
ConstraintValidator has it as a string and it extends it